tooling/build/scripts/publishing/index.ts

```diff
@@ -139,6 +146,8 @@ async function main() {
       }
     }
 
+    logDebug("Sitemap entries:", sitemapEntries)
```

Comment:
is this to be removed?

Reply:
This is `logDebug`, so it only appears if `DEBUG` is set to `true`. Keeping this so we can have a way to understand how the sitemap looks like during the build process.